src/base/rss/rss_autodownloader.cpp

```diff
@@ -248,25 +243,23 @@ void AutoDownloader::processJob(const QSharedPointer<ProcessingJob> &job)
         m_dirty = true;
         storeDeferred();
 
-        BitTorrent::AddTorrentParams params;
-        params.savePath = rule.savePath();
-        params.category = rule.assignedCategory();
-        params.addPaused = rule.addPaused();
-        BitTorrent::Session::instance()->addTorrent(job->torrentURL, params);
+        if (auto feed = Session::instance()->feedByURL(job->feedURL)) {
+            if (auto article = feed->articleByGUID(job->articleGUID)) {
```

Comment:
The auto downloading jobs are performed deferred, so article as well as its feed might not exist at this point, but we still have to handle the job.
Please, try first to understand how it works.